envbuilder.go

```diff
@@ -968,11 +1003,11 @@ func DefaultWorkspaceFolder(repoURL string) (string, error) {
 	if repoURL == "" {
 		return "/workspaces/empty", nil
 	}
-	parsed, err := url.Parse(repoURL)
-	if err != nil {
-		return "", err
-	}
-	name := strings.Split(parsed.Path, "/")
+
+	reg := regexp.MustCompile(`.*/`)
+	parsed := reg.ReplaceAllString(repoURL, "")
```

Comment:
```go
	parsed := repoURL
	if _, after, ok := strings.Cut(repoURL, "/"); ok {
		parsed = after
	}
```

Reply:
This is a good way to extract the repository name like `git@github.com:<username>/<repository name>.git` (github repository), but it doesn't work for gerrit repository url like `ssh://<username>@<gerrit-server>:<gerrit-ssh-port>/<repository name>`